### ISSUE TITLE
Items seem to not appearing in inventory

### ISSUE THREAD

**Issue by BudderSticks** [iTXTech/Genisys#2109]
I found a very annoying glitch with Genisys MCPE 0.15.10 (API 2.0.0 PHP7). The glitch is when items are supposed to appear in your inventory they do not. But when you die they drop. This error occurs when you use any sort of method that requires items to appear in inventory.

Genisys: 2.0.0
PHP: 7
Server OS: is Pocketmine I believe
Game version: iOS MCPE

Reproducing Glitch:

It is very easy to reproduce the glitch. All the player needs to do is for example do /give player apple 1 The apple will not appear in inventory. Or whenever addItem method is called the items will not appear.

How Glitch Started:

I believe the glitch started when my Genisys updated to 0.15.1 It may have something to do with my plugins, but I have found no faulty plugins.

Plugin List:

-BuildUHCJericMD
-EasyMessages
-Functions
-Genisys
-HereAuth
-MyPlot
-PureChat
-PurePerms
-SW_svile
-SkyBlock
-SwaggyMOTD
-TNTRun
-WorldEdit
-WorldGM
-essentialsTP
-iProtecter

Need anything else like OS or jt anything send me a message and I will be glad to send em over.

**Comment by poustmal** [comment]
How items drop on death on your server?I have problem with this.

**Comment by poustmal** [comment]
And you are gm 0 when you have items?

**Comment by BudderSticks** [comment]
Yes, and it seems the items are invisible in your inventory because when you tap the ground with open hand you get item.

**Comment by IrishPacks** [comment]
OS is Operating System which is not Genisys.

**Comment by BudderSticks** [comment]
Yes im gm 0 and it seems the items are invisible. So when you die they drop

**Comment by BudderSticks** [comment]
@IrishPacks Sorry XD

**Comment by dktapps** [comment]
This is a known bug from the Windows 10 Update.

**Comment by dktapps** [comment]
I'm talking about Genisys' Windows 10 update. -_-

**Comment by BudderSticks** [comment]
Okay, undterstood. Any ideas how to fix this? Plus random small issue i cant get snow off my server soawn i break it and then when i re join it re spawns. Disabled weather too

**Comment by starterdragon** [comment]
so this is a genisys problem, that is why some players don't get pre-made kits on ArcherPVP, BUHC and SkyWars

**Comment by Enzoomet** [comment]
Hi there! The issue has been forwarded to the developers. Expect a response from them soon.

**Comment by Muqsit** [comment]
I think a developer has already responded...

**Comment by ImagicalGamer** [comment]
@BudderSticks the snow issue is most likely a chunk loading problem if the biome is a snowy biome (not sure the ID or name for the official snowy biome).

**Comment by BudderSticks** [comment]
so ill try /biome 1
